pyfar/dsp/dsp.py

```diff
@@ -811,6 +813,338 @@ def regularized_spectrum_inversion(
     return inverse
 
 
+class interpolate_spectrum():
+    """
+    Interpolate incomplete spectrum to complete single sided spectrum.
+
+    TODO: Add reference to minimum phase functions
+
+    Parameters
+    ----------
+    data : FrequencyData
+        Input data to be interpolated. `data.fft_norm` must be `'none'`.
```

Comment:
Maybe add some remarks on this being supposed to interpolate TFs

Reply:
Yes, good idea. I added a remark below the one line doc string.